src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20230920145213_EES4477_MakeContactTelNosNullable.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES4477_MakeContactTelNosNullable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
+                name: "ContactTelNo",
+                table: "Contacts",
+                type: "nvarchar(max)",
+                nullable: true,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(max)");
+
+            migrationBuilder.Sql(
+                @"UPDATE [dbo].[Contacts]
+                  SET ContactTelNo = NULL
+                  WHERE TRIM(ContactTelNo) = '03700002288'
+                  OR TRIM(ContactTelNo) = '0370 000 2288'
+                  OR TRIM(ContactTelNo) = '037 0000 2288';");
```

Comment:
Was thinking `TRIM` is probably not necessary here, as strings are padded when compared for equality (try out `SELECT 'Equal!' WHERE '03700002288    ' = '03700002288'` but then I realised that it's only the case for trailing space, not leading space so you are probably right to include the trim here.